packages/react-reconciler/src/__tests__/useEvent-test.js

```diff
@@ -557,6 +557,30 @@ describe('useEvent', () => {
     expect(Scheduler).toHaveYielded(['Effect value: 2', 'Event value: 2']);
   });
 
+  // @gate enableUseEventHook
+  it("doesn't provide a stable identity", () => {
+    function Counter({shouldRender}) {
+      const onClick = useEvent(() => {
+        Scheduler.unstable_yieldValue('onClick');
+      });
+
+      // onClick doesn't have a stable function identity so this effect will fire on every render.
+      // In a real app useEvent functions should *not* be passed as a dependency, this is for
+      // testing purposes only.
+      useEffect(() => {
+        onClick();
+      }, [onClick]);
```

Comment:
We need a test where the event _is_ omitted as a dependency, but it still sees the latest values.